app/screens/index.tsx

```diff
@@ -211,6 +211,9 @@ Navigation.setLazyComponentRegistrator((screenName) => {
         case Screens.USER_PROFILE:
             screen = withServerDatabase(require('@screens/user_profile').default);
             break;
+        case Screens.CALL:
+            screen = withServerDatabase(require('@app/products/calls/screens/call_screen').default);
```

Comment:
I've tried, and for some reason `tsc` doesn't respect the `@calls` alias. :(  So instead of having all the red, I just removed the alias in the `babel.config` file. Happy to put it back in if you can figure out a way to make `tsc` happy!

Reply:
You need it in the babel config but also in the tsconfig